functions/functionsv2/firebase/upper/upper.go

```diff
@@ -65,7 +65,15 @@ func init() {
 // the `original` value of the document to upper case.
 func MakeUpperCase(ctx context.Context, e event.Event) error {
 	var data firestoredata.DocumentEventData
-	if err := proto.Unmarshal(e.Data(), &data); err != nil {
+
+	// If you omit `DiscardUnknown`, protojson.Unmarshal returns an error
+	// when encountering a new or unknown field.
+	options := proto.UnmarshalOptions{
+		DiscardUnknown: true,
+	}
+	err := options.Unmarshal(e.Data(), &data)
+
+	if err != nil {
```

Comment:
Thanks for also moving `if err != nil` into its own line.

Reply:
👍🏻